components/Card/card.css

```diff
@@ -0,0 +1,154 @@
+:root {
+    --card-hover-color: rgb(243, 241, 241);
+    --card-text-color: rgb(128, 128, 128);
+    --bg-color: rgb(255, 255, 255);
+    --bg-color2: rgb(56, 142, 60);
```

Comment:
Use a different name for two same variables instead of number.

Reply:
done